frontend/src/pages/reports/CurationReports.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useState } from "react";
+import styles from "./CurationReports.module.css"
+import { Dropdown, Label, ResponsiveMode, Spinner } from "@fluentui/react";
+import { IconFilePlus, IconX } from "@tabler/icons-react";
+import { createReport, getFilteredReports } from "../../api";
+
+const CurationReports  = () => {
+
+    const [dataLoad, setDataLoad] = useState(false)
+    const [filteredReports, setFilteredReports] = useState<any[]>([]);
+    const [loading, setLoading] = useState(false);
+    const [isCreateModalOpen, setIsCreateModalOpen] = useState(false);
+    const [isConfirm, setIsConfirm] = useState(false);
+    const [inputReportName, setinputReportName] = useState('')
+    const typeOptions = [
```

Comment:
@egdagger theses are no types these are curation reports they should be pre loaded in the database and this UI should allow the user to create new reports or delete reports 

curation report’s doesn’t have a type